integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/Pages/MetalakePage.java

```diff
@@ -0,0 +1,295 @@
+  }
+
+  //  public WebElement findElementByLink(String name) {
+  //    String xpath = "//div[@data-field='name']//a[@href='/ui/metalakes?metalake=" + name + "']";
+  //
+  //    return driver.findElement(By.xpath(xpath));
+  //  }
+
+  public void setMetalakeNameField(String nameField) {
+    metalakeNameField.sendKeys(
+        Keys.chord(Keys.HOME, Keys.chord(Keys.SHIFT, Keys.END), Keys.DELETE));
+    metalakeNameField.clear();
+    metalakeNameField.sendKeys(nameField);
+  }
+
+  public void setMetalakeCommentField(String commentField) {
+    metalakeCommentField.sendKeys(
+        Keys.chord(Keys.HOME, Keys.chord(Keys.SHIFT, Keys.END), Keys.DELETE));
+    metalakeCommentField.clear();
+    metalakeCommentField.sendKeys(commentField);
+  }
+
+  public void setQueryInput(String queryInput) {
+    clearQueryInput();
+    queryMetalakeInput.sendKeys(queryInput);
+  }
+
+  public void clearQueryInput() {
+    queryMetalakeInput.sendKeys(
+        Keys.chord(Keys.HOME, Keys.chord(Keys.SHIFT, Keys.END), Keys.DELETE));
+    queryMetalakeInput.clear();
+  }
+
+  public void clickDeleteMetalakeBtn(String name) {
+    String xpath = "//button[@data-refer='delete-metalake-" + name + "']";
+    WebElement deleteMetalakeBtn = driver.findElement(By.xpath(xpath));
+    deleteMetalakeBtn.click();
+  }
+
+  public void clickViewMetalakeBtn(String name) {
+    String xpath = "//button[@data-refer='view-metalake-" + name + "']";
+    WebElement viewMetalakeBtn = driver.findElement(By.xpath(xpath));
+    viewMetalakeBtn.click();
+  }
+
+  public void clickEditMetalakeBtn(String name) {
+    String xpath = "//button[@data-refer='edit-metalake-" + name + "']";
+    WebElement editMetalakeBtn = driver.findElement(By.xpath(xpath));
+    editMetalakeBtn.click();
+  }
+
+  public void clickMetalakeLink(String name) {
+    String xpath = "//a[@href='/ui/metalakes?metalake=" + name + "']";
+    WebElement metalakeLink = driver.findElement(By.xpath(xpath));
+    WebDriverWait wait = new WebDriverWait(driver, 10);
+    wait.until(ExpectedConditions.elementToBeClickable(By.xpath(xpath)));
+    metalakeLink.click();
+  }
+
+  public void setMetalakeProps(int index, String key, String value) {
+    // Set the indexed props key
+    String keyPath = "//div[@data-refer='add-props-key-" + index + "']//input[@name='key']";
+    WebElement keyInput = driver.findElement(By.xpath(keyPath));
+    keyInput.sendKeys(key);
+
+    // Set the indexed props value
+    String valuePath = "//div[@data-refer='add-props-value-" + index + "']//input[@name='value']";
+    WebElement valueInput = driver.findElement(By.xpath(valuePath));
+    valueInput.sendKeys(value);
+  }
+
+  public boolean checkIsErrorName() {
+    String xpath = "//div[@data-refer='metalake-name-field']";
+    WebElement nameField = findElementByXPath(xpath);
+    List<WebElement> errorText =
+        nameField.findElements(By.xpath("//div[contains(@class, 'Mui-error')]"));
+
+    return !errorText.isEmpty();
```

Comment:
Please use `Strings.isEmpty()` function.

Reply:
This is the element function.